drivers/sensors/pmw3360.c

```diff
@@ -86,23 +89,18 @@ void print_byte(uint8_t byte) { dprintf("%c%c%c%c%c%c%c%c|", (byte & 0x80 ? '1'
 #endif
 #define constrain(amt, low, high) ((amt) < (low) ? (low) : ((amt) > (high) ? (high) : (amt)))
 
-bool spi_start_adv(void) {
-    bool status = spi_start(PMW3360_CS_PIN, PMW3360_SPI_LSBFIRST, PMW3360_SPI_MODE, PMW3360_SPI_DIVISOR);
+void pmw3360_spi_start(void) {
+    spi_start(PMW3360_CS_PIN, PMW3360_SPI_LSBFIRST, PMW3360_SPI_MODE, PMW3360_SPI_DIVISOR);
     wait_us(1);
-    return status;
 }
 
-void spi_stop_adv(void) {
-    wait_us(1);
```

Comment:
IIRC, these waits are needed to help ensure timing.

Reply:
It works fine on the 2 (identical) sensors that I tried. I currently don't have access to my ploopy mouse and will verify this again there in January, as I think more of the sleeps can go away.

If you read the diff carefully, this spi_stop_adv is used only 3 times.
-  at the end of pmw3360_read_burst, where it is redundant with a call a few lines above
- in init, where it's sandwiched between even more sleeps: 
```
@@ -150,7 +146,7 @@ bool pmw3360_init(void) {
 
     pmw3360_spi_start();
     wait_us(40);
-    pmw3360_spi_stop();
+    spi_stop();
     wait_us(40);
```
(the whole start/stop dance looks suspicious anyway).
And finally the last place after firmware upload ... where there's another sleep just afterwards.

```
     pmw3360_upload_firmware();
 
-    pmw3360_spi_stop();
+    spi_stop();
 
     wait_ms(10);

```

IOW, I think that 1us sleep is redundant in all cases where it's being used.